docs/src/__examples__/ButtonGroup/DEFAULT.tsx

```diff
@@ -18,9 +18,4 @@ export default {
       </Button>
     </ButtonGroup>
   ),
-  info: {
-    title: "Default button group",
-    description:
-      "Button groups visually join their children to show related actions as a single unit.",
-  },
 };
```

Comment:
Sandbox showing examples without knobs should not display the button "Playground".